yb-voyager/cmd/conflictDetectionCache.go

```diff
@@ -96,21 +148,23 @@ func (c *ConflictDetectionCache) eventsConfict(event1, event2 *tgtdb.Event) bool
 
 		For now, we just check if the event is from same table then we consider it as a conflict
 	*/
-	if isTargetDBExporter(event2.ExporterRole) {
-		log.Infof("conflict detected for table %s, between event1(vsn=%d) and event2(vsn=%d)", event1.TableName, event1.Vsn, event2.Vsn)
+	if isTargetDBExporter(incomingEvent.ExporterRole) {
+		log.Infof("conflict detected for table %s, between event1(vsn=%d) and event2(vsn=%d)", cachedEvent.TableName, cachedEvent.Vsn, incomingEvent.Vsn)
 		return true
 	}
```

Comment:
For events exported from target stall the pipeline...
- if the cachedEvent is DELETE; or
- if the cachedEvent is UPDATE and both the cached and incoming events are modifying same unique column (without actually checking their values).

Reply:
This is not totally correct: assume if there are two uniq keys UK1 and UK2; and if `cachedEvent` is modifying `UK1` and `incomingEvent` is modifying `UK2`, the code incorrectly assumes a conflict. The correct condition should be:
```
ukList := lo.Intersect(cachedEventCols, uniqueKeyColumns)
if lo.Some(incomingEventCols, ukList) {
  conflict = true
}
```